modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -55,20 +61,20 @@ All Mule instances access the same Mule file folders concurrently, which can lea
 All Mule instances get the same TCP requests and then process duplicate messages.
 
 * JMS topics. +
-All Mule instances connect to the same JMS topic, which might lead to the repeated processing of messages when scaling the non-clustered Mule instance horizontally.
+All Mule instances connect to the same JMS topic, which might lead to the repeated processing of messages when scaling the non-clustered Mule instance horizontally. To the JMS broker the instances appear as separate subscribers and will all get a copy of each message, which is rarely the required behavior. Although there is a "shared subscriber" configuration available which might also address this requirement to instruct the JMS broker to treat all instances as a combined subscriber and to give them each separate messages.
 
 * JMS request-reply/request-response. +
 All Mule instances are listening for messages in the same response queue. This implies that a Mule instance might obtain a response that isn't correlated to the request it sent and might result in incorrect responses or make a flow fail with timeout.
 
-* Idempotent Redelivery Policy. +
-Idempotency doesn’t work if the same request is received by different Mule instances. Duplicated messages aren’t possible.
+* Idempotent Redelivery Policy and Idempotent Message Validation. +
+Idempotency doesn’t work correctly with horizontal scaling if the same request is received by different Mule instances if the Object Store contents used by the Redelivery policy or the Idempotency Message Validator is localized. With the cluster sharing the Object Store values used by these idepotency features - duplicated messages aren’t possible because all nodes are sharing the list of already processed identifiers.
```

Comment:
```suggestion
Idempotency doesn’t work correctly with horizontal scaling if the same request is received by different Mule instances and the Object Store contents used by the Redelivery policy or the Idempotency Message Validator is localized. For a cluster sharing Object Store values used by these idempotency features, duplicate messages aren’t possible because all nodes are sharing the list of already-processed identifiers.
```